vdev/src/commands/meta/starship.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Args;
+use std::fs::File;
+use std::io::{prelude::*, BufReader};
+use std::path::PathBuf;
+
+use crate::app::Application;
+
+/// Custom Starship prompt plugin
+#[derive(Args, Debug)]
+#[command(hide = true)]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(&self, app: &Application) {
+        let mut contexts = vec![];
+
+        let path: PathBuf = [&app.path, "Cargo.toml"].iter().collect();
+        if let Ok(file) = File::open(path) {
+            let reader = BufReader::new(file);
+
+            for line in reader.lines() {
+                if let Ok(line) = line {
+                    if line.starts_with("version = ") {
+                        contexts.push(format!("version: {}", &line[11..line.len() - 1]));
+                        break;
+                    }
+                }
+            }
+        };
```

Comment:
If grows to look up any more than a one-line value, I would suggest you pull in a TOML parser and get at the structured data instead of hand-rolling your own lookups.

Reply:
Will do! I just did this for a speedy prompt